pycqed/analysis/measurement_analysis.py

```diff
@@ -47,15 +49,30 @@
 class MeasurementAnalysis(object):
 
     def __init__(self, TwoD=False, folder=None, auto=True,
-                 cmap_chosen='viridis', **kw):
+                 cmap_chosen='viridis', no_of_columns=1, qb_name=None, **kw):
+
         if folder is None:
             self.folder = a_tools.get_folder(**kw)
         else:
             self.folder = folder
         self.load_hdf5data(**kw)
         self.fit_results = []
-        self.box_props = dict(boxstyle='Square', facecolor='white', alpha=0.8)
         self.cmap_chosen = cmap_chosen
+        self.no_of_columns = no_of_columns
+        self.dpi = 600                  #dpi for plots
```

Comment:
I don't think hardcoding these here makes sense. 
It would be much better to use `kw.get(default_value, 'dict_key')` instead. 

Even better would be some fancy system (like in ma_v2) but let's not worry abou that now. 

Not that I do think that keeping it in the same place for all analyses (like done here) is a very good idea and definitely an improvement 👍

Reply:
I fixed this in PR #341. Here I will leave this unchanged so there is no overlap with the other PR.